lib/parallel_tests/cli.rb

```diff
@@ -26,8 +26,13 @@ def run(argv)
 
     def execute_in_parallel(items, num_processes, options)
       Tempfile.open 'parallel_tests-lock' do |lock|
+        progress_indicator = Thread.new { while true do sleep 60; print '.' end } if options[:serialize_stdout]
         return Parallel.map(items, :in_threads => num_processes) do |item|
           result = yield(item)
+          if options[:serialize_stdout] && progress_indicator.alive?
+            progress_indicator.exit
+            puts
```

Comment:
Extra puts is just so that the first line of test output doesn't end up on the same line as the dots.